features/support/fixtures.rb

```diff
@@ -61,6 +61,37 @@ def self.user_research_participants_service
     }
   end
 
+  def self.user_research_studios_service
+    return {
+      # `nil` values should be updated within the step when this fixture is used
+      id: nil,
```

Comment:
I just copied them from the database and pasted in the same order they were there. Now I ordered them :)